### ISSUE TITLE
Cannot install on Raspberry Pi 3B via docker-compose: error on data-importer

### ISSUE THREAD

**Issue by elchami89** [firefly-iii/firefly-iii#8168]
### Support guidelines

- [X] I've read the [support guidelines](https://github.com/firefly-iii/firefly-iii/blob/main/.github/support.md)

### I've found a bug and checked that ...

- [X] ... [the documentation](https://docs.firefly-iii.org/) does not mention anything about my problem
- [X] ... there are no open or closed issues that are related to my problem
- [ ] ... it's [definitely a Firefly III issue, not me](https://github.com/firefly-iii/firefly-iii/blob/main/.github/its_you_not_me.md)

### Description

I have a Raspberry Pi 3B with the raspios_armhf (32 bits) and when I try to install Firefly III following the [guide](https://docs.firefly-iii.org/firefly-iii/installation/docker/) I get the next error:

> docker compose -f docker-compose-importer.yml up -d              
[+] Running 1/4
 ⠹ db Pulling                                                                                                                                          2.1s 
 ✘ importer Error                                                                                                                                      2.1s 
 ⠹ cron Pulling                                                                                                                                        2.1s 
 ⠹ app Pulling                                                                                                                                         2.1s 
no matching manifest for linux/arm/v7 in the manifest list entries

Could you help me with that, I would use Firefly III with the Data Importer

### Debug information

docker compose -f docker-compose-importer.yml up -d              
[+] Running 1/4
 ⠹ db Pulling                                                                                                                                          2.1s 
 ✘ importer Error                                                                                                                                      2.1s 
 ⠹ cron Pulling                                                                                                                                        2.1s 
 ⠹ app Pulling                                                                                                                                         2.1s 
no matching manifest for linux/arm/v7 in the manifest list entries

**Comment by JC5** [comment]
It's right here.

https://hub.docker.com/r/fireflyiii/data-importer/tags

I don't know what to do about this.

**Comment by elchami89** [comment]
> It's right here.
> 
> https://hub.docker.com/r/fireflyiii/data-importer/tags
> 
> I don't know what to do about this.

I'm using the [alternative Docker Compose file](https://raw.githubusercontent.com/firefly-iii/docker/main/docker-compose-importer.yml) without changes

> importer:
    image: fireflyiii/data-importer:latest
    hostname: importer
    restart: always
    container_name: firefly_iii_importer
    networks:
      - firefly_iii
    ports:
      - '81:8080'
    depends_on:
      - app
    env_file: .importer.env

I thought it will take automatically the image

**Comment by JC5** [comment]
Yes so do I, I don't know what you want me to tell you. The correct tag is right there on Docker Hub.
